weed/storage/volume_vacuum_test.go

```diff
@@ -107,13 +106,15 @@ func testCompaction(t *testing.T, needleMapKind NeedleMapKind) {
 		if realWatermark != watermark {
 			t.Fatalf("testing watermark failed")
 		}
-		if int(deleteCount+v.FileCount()+v.DeletedCount()) != beforeCommitFileCount+afterCommitFileCount*2 {
+		// logic deletedcount is different between memmory and leveldb. So this check is not common.
```

Comment:
Is this comment still valid?

Reply:
Yes, it's valid. In memory scenario, the check is invalid.
Because the deleted count of statistical methord betweent memory and levelDB index is different too.
I used to try to modify them to make it stay same, but faild due to bloomfilter
So I check this only in leveldb scenario.
https://github.com/seaweedfs/seaweedfs/blob/3c8e95eeb33a97a094b6854ae928bbb1a767e1f6/weed/storage/needle_map_memory.go#L42-L50

Reply:
ok. Please explain it in the comment.

Reply:
In memory scenario if a file was deleted  many times, the DeletionCounter will be added only once because the code will check the old needle's size.

 In leveldb scenario if a file was deleted  many times, the DeletionCounter will be added every time, because the code doesn't check the old needle's size.

Bye the way, I'll remove the test for DeletionCounter because of the above reason.

Reply:
Here is the code for multipal deleting.
https://github.com/seaweedfs/seaweedfs/blob/676e27c589a99691ba78f010a067898d89f764ac/weed/storage/needle_map_metric.go#L110-L120

Reply:
@chrislusf Is this a bug and needs to be fixed? Or it doesn't matter?